stdlib/format.ml

```diff
@@ -646,35 +574,28 @@ let pp_get_formatter_tag_functions state () = {
 }
 
 
-let pp_set_formatter_tag_functions state {
-     mark_open_tag = mot;
-     mark_close_tag = mct;
-     print_open_tag = pot;
-     print_close_tag = pct;
-  } =
-  state.pp_mark_open_tag <- mot;
-  state.pp_mark_close_tag <- mct;
-  state.pp_print_open_tag <- pot;
-  state.pp_print_close_tag <- pct
+let pp_set_formatter_tag_functions state tag_functions =
```

Comment:
Here it is better to use pattern matching on the argument to make sure the compiler will let us know if fields are ever added to the argument type.